test/functional/apps/dashboard/index.js

```diff
@@ -23,6 +23,7 @@ export default function({ getService, loadTestFile }) {
 
   async function loadCurrentData() {
     await browser.setWindowSize(1300, 900);
+    await esArchiver.unload('logstash_functional');
```

Comment:
Is this really necessary? how often is this called?

Reply:
looks like it's just called once for ciGroup2 and once for ciGroup3